vmdb/app/models/vm_or_template.rb

```diff
@@ -328,7 +328,7 @@ def refresh_state
 
   def run_command_via_parent(verb, options = {})
     # TODO: Need to break this logic out into a method that can look at the verb and the vm and decide the best way to invoke it - Virtual Center WS, ESX WS, Storage Proxy.
-    if self.ext_management_system && self.ext_management_system.authentication_valid? && self.ext_management_system.respond_to?(verb)
+    if self.ext_management_system && self.ext_management_system.has_credentials? && self.ext_management_system.respond_to?(verb)
```

Comment:
This method is for actually running commands on the provider...I *think* it should be authentication_status_ok.  @gmcculloug ?

P.S. I hate this method :smile_cat: